src/mobile-header/mobile-header.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+import * as React from 'react';
+import { getOverrides } from '../helpers/overrides';
+import { Button, KIND, SHAPE } from '../button';
+import {
+  StyledRoot,
+  StyledNavContainer,
+  StyledTitle,
+  StyledActionButtonsContainer,
+} from './styled-components';
+import { TYPE } from './constants';
+import { useStyletron } from '../styles/index';
+import type { MobileHeaderProps } from './types';
+
+const renderButtonContent = (content) => {
```

Comment:
The design docs indicate that the content for the MobileHeader buttons can be either an icon or a string. My sense is that if the consumer wants to use an icon, the component would ideally size the icon appropriately. But what if they want to pass in something that is't an icon (or a string)? In that case this render function is passing in an erroneous `size` prop.